python/rsgislib/classification/classaccuracymetrics.py

```diff
@@ -175,7 +175,10 @@ class name.
                           a JSON file (Default=None).
     :param out_csv_file: if specified the generated metrics and confusion matrix are written to
                          a CSV file (Default=None).
-
+    Example::
+    
```

Comment:
or would you prefer:

vector_file = "Sonoma_County_RF_classification_ref_samples.gpkg"
ref_col ="classification"
etc...

rsgislib.classification.calc_acc_metrics_vecsamples(vector_file, ref_col, ...)